pipeline/transformer/transformer_sierra/src/main/scala/uk/ac/wellcome/platform/transformer/sierra/transformers/SierraAlternativeTitles.scala

```diff
@@ -25,5 +29,12 @@ object SierraAlternativeTitles extends SierraTransformer with SierraQueryOps {
       .filterNot { varfield =>
         varfield.marcTag.contains("246") && varfield.indicator2.contains("6")
       }
-      .flatMap(_.subfields.contentString(" "))
+      .flatMap { varfield =>
+        varfield.subfields
+          .filter {
+            case MarcSubfield("5", "UkLW") => false
+            case _                         => true
+          }
+          .contentString(" ")
+      }
```

Comment:
This is doing two steps at once: throwing away the bad subfield, then joining them with `contentString(" ")`. Is it worth splitting into two distinct steps?

```suggestion
      .flatMap { varfield =>
        varfield.copy(subfields = 
          varfield.subfields
            .filter {
              case MarcSubfield("5", "UkLW") => false
              case _                         => true
            }
        )
      }
      .flatMap(_.subfields.contentString(" "))
```

Reply:
Not sure about this - I find it less readable to be honest

Reply:
I'd err on the side of @jamieparkinson's code here. As much as it could be seen as two steps, you could see it as throwing away the bad subfield and joining with `" "` as 2 step.

Reply:
🚲 🏠